### ISSUE TITLE
Show `blst` hardware support in `lighthouse --version`

### ISSUE THREAD

**Issue by michaelsproul** [sigp/lighthouse#5918]
## Description

Now that we're using `portable` `blst` everywhere, we could show the hardware acceleration support _dynamically_ in `lighthouse --version`, like we do for SHA256.

Something like:

```
Lighthouse v5.2.0-f1d88ba
BLS library: blst-portable
BLS hardware acceleration: true
SHA256 hardware acceleration: false
Allocator: jemalloc
Profile: maxperf
Specs: mainnet (true), minimal (true), gnosis (true)
```

**Comment by scafe3** [comment]
I would like to have a try.

**Comment by michaelsproul** [comment]
Great, thanks @scafe3

**Comment by chong-he** [comment]
Closed by #6039
